### ISSUE TITLE
Ability to downscale/resize to smaller screen size

### ISSUE THREAD

**Issue by Lreaper** [mihonapp/mihon#550]
### Describe your suggested feature

As the title suggests, having the possibility to zoom out further than 100% would be beneficial if you're reading HQ manga on a larger tablet. Currently, if you're reading manga that has high resolution images these images will fill the whole tablet size, making the pages appear bigger than they should while also making the overall quality less crisp. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.4](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by BrutuZ** [comment]
Have you properly set the reader's [scale type](https://mihon.app/docs/guides/reader-settings#scale-type)?

**Comment by Lreaper** [comment]
> Have you properly set the reader's [scale type](https://mihon.app/docs/guides/reader-settings#scale-type)?

Yes, but unfortunately none of the available options allow you to go below the images' resolution scale

**Comment by Lreaper** [comment]
I have created a mock up visual representation of what I have tried to express in the opening post.

Let's say you're reading manga that has high DPI images, these images will most probably have high resolutions, so if you Fit to Width, you get something like:

![fit width](https://github.com/mihonapp/mihon/assets/19478763/e9aaa369-82f5-4a9e-b9ca-c4f91fd8ad62)

This is fine, and still looks decent on a larger tablet, but the page size will end up being much bigger than its intended, or the actual size of the original material.

What I mean by downscaling is, keeping the image's dpi, but reducing the "viewport" to something that more closely matches the original material:

![downscale](https://github.com/mihonapp/mihon/assets/19478763/4a01b82f-0b8e-4339-9639-62f3fcb3b5b3)

So the image itself would still be high res, and if you wish to manually zoom in, you would get the benefits of having a high dpi image, but this would keep the actual display size more closer to the original.
